src/components/Book.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { removeBook } from '../redux/books/books';
+
+const Book = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+
+  const deleteBook = (event) => {
+    event.preventDefault();
+    dispatch(removeBook(id));
+  };
+
+  return (
+    <>
+      <div className="book-item">
+        <div>
+          <h2 className="book-title">{title}</h2>
+          <p className="">{author}</p>
+          <div className="links-container">
+            <ul>
+              <li><button type="button" className="btn-1 line">Comments</button></li>
+              <li><button type="button" className="btn-1 line">Edit</button></li>
+              <li><button type="button" className="btn-1" onClick={deleteBook} value={id}>Remove</button></li>
+            </ul>
+          </div>
+        </div>
+        <div className="progress">
+          <CircularProgressbar
+            className="progressBar"
+            variant="determinate"
+            color="primary"
+            sx={{
+              width: 50,
+            }}
+            value={50}
+          />
+          <div className="numerics">
+            <span className="number">50%</span>
+            <span className="completed">completed</span>
+          </div>
+        </div>
+        <div className="view-progress">
+          <span className="chapter-title">Current chapter</span>
+          <h3 className="chapter-number">Chapter 10</h3>
+          <button type="button" className="btn-2">
+            Update progress
+          </button>
+        </div>
+      </div>
+    </>
```

Comment:
- [ ] Kindly ensure your styling matches the design exactly. You are missing the left and right margins around be body of the book cards as we have in the design.

- [ ] The subtext on the left should have a reduced size and change the color to the blue color we have in the design.

- [ ] The content of the right section after the demarcation in the book cards should be moved to the left as we have in the design.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![owau](https://user-images.githubusercontent.com/54848363/222683118-82347701-d7a1-4349-a41d-01c020acc656.jpg) | ![dioq](https://user-images.githubusercontent.com/54848363/222683103-5eab4460-5a70-4449-b8be-1be1923888f5.jpg)